packages/vaadin-charts/vaadin-chart-base-theme.js

```diff
@@ -0,0 +1 @@
+import './theme/vaadin-chart-base-theme.js';
```

Comment:
Is this new root-level file necessary?

Reply:
No, it is not. It was removed in another commit in this PR.

Reply:
Another commit removed the file from `@vaadin/charts`. This one should be removed also

Reply:
Ah, the other package. You're right. It is removed now.